### PR TITLE
Add matrix-consts as argument

### PR DESCRIPTION
**Pull request by mor-m-shemesh**

### CODE REVIEW

File: src/corona_hakab_model/generation/matrix_consts.py [hunk 1]
```diff
@@ -1,26 +1,60 @@
-from typing import NamedTuple
-
+from collections import namedtuple
```
Comment (bentheiii) [inline src/corona_hakab_model/generation/matrix_consts.py:1]: this file should look like consts.py

File: src/corona_hakab_model/main.py [hunk 2]
```diff
@@ -2,56 +2,60 @@
...
-    generate_parser.add_argument("--circles", dest="circles_consts_path", help="Parameter file with consts for the circles")
+    parser.add_argument("--simulation-parameters",
+                        dest="simulation_parameters_path",
+                        help="Parameters for simulation engine")
+    parser.add_argument("--circles-consts",
+                        dest="circles_consts_path",
+                        help="Parameter file with consts for the circles")
+    parser.add_argument("--matrix-consts",
+                        dest="matrix_consts_path",
+                        help="Parameter file with consts for the matrix")
```
Comment (bentheiii) [inline src/corona_hakab_model/main.py:39]: give short names for conveniance

File: src/corona_hakab_model/generation/matrix_consts.py (hunk 1 above)
Comment (mor-m-shemesh) [reply src/corona_hakab_model/generation/matrix_consts.py:1]: how does it NOT look like consts.py?

Comment (mor-m-shemesh) [reply src/corona_hakab_model/generation/matrix_consts.py]: Done :)

File: src/corona_hakab_model/main.py (hunk 2 above)
Comment (mor-m-shemesh) [reply src/corona_hakab_model/main.py]: Done :)

**Comment by RoyIsland** [comment]
רק חסר לעשות override ל__hash__  ו-__eq__  גם לCirclesConsts

**Pull request merged**